2014/10/12 15:31:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/?r=gii
---
2014/10/12 15:32:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=gii
---
2014/10/12 16:21:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "create".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('create')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casoUso/create')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/projman/index.php?r=casoUso/create
---
2014/10/12 16:35:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "permissaos".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('permissaos')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=permissaos
---
2014/10/12 16:41:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "AlteracoesRequisito".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('AlteracoesRequi...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=AlteracoesRequisito
---
2014/10/12 16:42:02 [error] [exception.CException] exception 'CException' with message 'AlteracoesRequisitoController cannot find the requested view "index".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#1 C:\xampp\htdocs\projman\protected\controllers\AlteracoesRequisitoController.php(11): CController->render('index', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): AlteracoesRequisitoController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('AlteracoesRequi...')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/projman/index.php?r=AlteracoesRequisito
---
2014/10/12 16:48:28 [error] [exception.CException] exception 'CException' with message 'The attribute must be specified in the format of "Name:Type:Label", where "Type" and "Label" are optional.' in C:\xampp\htdocs\yii\framework\zii\widgets\CDetailView.php:183
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(173): CDetailView->run()
#1 C:\xampp\htdocs\projman\protected\views\casoUso\view.php(33): CBaseController->widget('zii.widgets.CDe...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\projman\protected\controllers\CasoUsoController.php(56): CController->render('view', Array)
#7 [internal function]: CasoUsoController->actionView('1')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(CasoUsoController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(CasoUsoController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casoUso/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/projman/index.php?r=casoUso/view&id=1
HTTP_REFERER=http://localhost/projman/index.php?r=casoUso/create
---
2014/10/12 16:49:45 [error] [exception.CException] exception 'CException' with message 'The attribute must be specified in the format of "Name:Type:Label", where "Type" and "Label" are optional.' in C:\xampp\htdocs\yii\framework\zii\widgets\CDetailView.php:183
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(173): CDetailView->run()
#1 C:\xampp\htdocs\projman\protected\views\casoUso\view.php(33): CBaseController->widget('zii.widgets.CDe...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\projman\protected\controllers\CasoUsoController.php(56): CController->render('view', Array)
#7 [internal function]: CasoUsoController->actionView('2')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(CasoUsoController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(CasoUsoController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casoUso/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/projman/index.php?r=casoUso/view&id=2
HTTP_REFERER=http://localhost/projman/index.php?r=casoUso/create
---
2014/10/12 16:54:29 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`root`.`ecra_caso`, CONSTRAINT `ecra_caso_ibfk_1` FOREIGN KEY (`ecra`) REFERENCES `ecra` (`cod_ecra`)). The SQL statement executed was: UPDATE `ecra_caso` SET `ecra`=:yp0, `caso_uso`=:yp1 WHERE `ecra_caso`.`ecra`=1 AND `ecra_caso`.`caso_uso`=1.
in C:\xampp\htdocs\projman\protected\controllers\EcraCasoController.php (130)
in C:\xampp\htdocs\projman\protected\controllers\EcraCasoController.php (90)
in C:\xampp\htdocs\projman\index.php (13)
2014/10/12 16:54:29 [error] [exception.CHttpException.23000] exception 'CHttpException' with message 'This operation is not permitted due to an existing foreign key reference.' in C:\xampp\htdocs\projman\protected\controllers\EcraCasoController.php:144
Stack trace:
#0 C:\xampp\htdocs\projman\protected\controllers\EcraCasoController.php(134): EcraCasoController->showError(Object(CDbException))
#1 C:\xampp\htdocs\projman\protected\controllers\EcraCasoController.php(90): EcraCasoController->saveModel(Object(EcraCaso))
#2 [internal function]: EcraCasoController->actionUpdate('1', '1')
#3 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(EcraCasoController), Array)
#4 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(EcraCasoController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ecracaso/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#11 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/projman/index.php?r=ecracaso/update&ecra=1&caso_uso=1
HTTP_REFERER=http://localhost/projman/index.php?r=ecracaso/update&ecra=1&caso_uso=1
---
2014/10/20 11:47:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "GII".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('GII')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=GII
---
2014/10/20 11:48:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/?r=user
---
2014/10/20 11:48:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/?r=user
---
2014/10/20 11:54:11 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\models\LoginForm.php (52)
2014/10/20 11:54:11 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=site/login
HTTP_REFERER=http://localhost/projman/index.php?r=site/login
---
2014/10/20 11:58:27 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\models\LoginForm.php (52)
2014/10/20 11:58:27 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=site/login
HTTP_REFERER=http://localhost/projman/index.php?r=site/login
---
2014/10/20 11:59:54 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\models\LoginForm.php (52)
2014/10/20 11:59:54 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=site/login
HTTP_REFERER=http://localhost/projman/index.php?r=site/login
---
2014/10/20 12:03:23 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\models\LoginForm.php (52)
2014/10/20 12:03:23 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=site/login
HTTP_REFERER=http://localhost/projman/index.php?r=site/login
---
2014/10/20 12:03:31 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\models\LoginForm.php (52)
2014/10/20 12:03:31 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=site/login
HTTP_REFERER=http://localhost/projman/index.php?r=site/login
---
2014/10/20 12:03:35 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\models\LoginForm.php (52)
2014/10/20 12:03:35 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=site/login
HTTP_REFERER=http://localhost/projman/index.php?r=site/login
---
2014/10/20 12:05:55 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\models\LoginForm.php (52)
2014/10/20 12:05:55 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=site/login
HTTP_REFERER=http://localhost/projman/index.php?r=site/login
---
2014/10/24 11:21:18 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 11:26:30 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 11:42:03 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:00:47 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:01:02 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.isAdmin" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('isAdmin')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CWebUser->__get('isAdmin')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:02:22 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.users" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('users')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CModule->__get('users')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:02:33 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.isAdmin" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('isAdmin')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CWebUser->__get('isAdmin')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:03:08 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.superuser" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('superuser')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CWebUser->__get('superuser')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:03:22 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.tbl_users" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('tbl_users')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CModule->__get('tbl_users')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->tbl...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:03:33 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.superuser" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('superuser')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CWebUser->__get('superuser')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:07:56 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.superuser" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('superuser')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CWebUser->__get('superuser')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->Use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:09:20 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:23:48 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.isAdmin" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('isAdmin')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CWebUser->__get('isAdmin')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/?r=casouso
---
2014/10/24 12:24:25 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.isAdmin" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(146): CComponent->__get('isAdmin')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(612) : eval()'d code(1): CWebUser->__get('isAdmin')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(612): eval()
#3 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(392): CComponent->evaluateExpression('Yii::app()->Use...', Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(288): CAccessRule->isExpressionMatched(Object(CWebUser))
#5 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(138): CAccessRule->isUserAllowed(Object(CWebUser), Object(CasoUsoController), Object(CInlineAction), '::1', 'GET')
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/?r=casouso
---
2014/10/24 12:28:59 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso
---
2014/10/24 12:29:16 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:29:33 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:31:10 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:31:25 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:31:39 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:32:02 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:32:12 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:36:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "home/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=home/index
---
2014/10/24 12:37:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "home/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=home/index
---
2014/10/24 12:37:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/index.p...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/projman/index.php?r=casouso/index.php
---
2014/10/24 12:37:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/index.p...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/projman/index.php?r=casouso/index.php
---
2014/10/24 12:38:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "../index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('../index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=../index
---
2014/10/24 12:39:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "../index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('../index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=../index
---
2014/10/24 12:39:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "../index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('../index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=../index
---
2014/10/24 12:50:06 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 12:50:29 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:06:26 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 13:09:27 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casoUso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casoUso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casoUso/index
---
2014/10/24 13:09:41 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casoUso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casoUso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casoUso/index
---
2014/10/24 13:10:03 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:10:14 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 13:10:23 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 13:10:52 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/create
---
2014/10/24 13:12:01 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:21:35 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:22:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "alteracaorequisito/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracaorequis...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=alteracaorequisito/index
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:23:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "alteracaorequisito/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracaorequis...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=alteracaorequisito/index
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:23:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "alteracaorequisito/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracaorequis...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projman/index.php?r=alteracaorequisito/index
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:30:18 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracao/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=alteracao/admin
HTTP_REFERER=http://localhost/projman/index.php?r=alteracao/index
---
2014/10/24 13:30:29 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/24 13:30:31 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=cliente/admin
HTTP_REFERER=http://localhost/projman/index.php?r=cliente/index
---
2014/10/24 13:30:34 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ecra/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=ecra/admin
HTTP_REFERER=http://localhost/projman/index.php?r=ecra/index
---
2014/10/24 13:30:37 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ecra/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=ecra/admin
HTTP_REFERER=http://localhost/projman/index.php?r=ecra/index
---
2014/10/24 13:35:59 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracao/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=alteracao/admin
HTTP_REFERER=http://localhost/projman/index.php?r=alteracao/index
---
2014/10/24 13:36:07 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso
---
2014/10/24 13:38:30 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracao/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=alteracao/admin
HTTP_REFERER=http://localhost/projman/index.php?r=alteracao/index
---
2014/10/24 13:41:44 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracao/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=alteracao/admin
HTTP_REFERER=http://localhost/projman/index.php?r=alteracao/index
---
2014/10/24 13:42:48 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/10/29 17:59:34 [error] [exception.CDbException] exception 'CDbException' with message 'Relation "pessoa" is not defined in active record class "Requisitos".' in C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php:226
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(296): CActiveFinder->buildJoinTree(Object(CJoinElement), 'pessoa')
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(48): CActiveFinder->buildJoinTree(Object(CJoinElement), Array)
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(909): CActiveFinder->__construct(Object(Requisitos), Array)
#3 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1604): CActiveRecord->getActiveFinder(Array)
#4 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#5 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#6 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#7 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#8 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#9 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(339): CBaseListView->init()
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#12 C:\xampp\htdocs\projman\protected\views\requisitos\admin.php(68): CBaseController->widget('zii.widgets.gri...', Array)
#13 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#17 C:\xampp\htdocs\projman\protected\controllers\RequisitosController.php(144): CController->render('admin', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): RequisitosController->actionAdmin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#22 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#23 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#24 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#25 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#26 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#27 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#28 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/admi...')
#29 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#30 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#31 {main}
REQUEST_URI=/projman/index.php?r=requisitos/admin
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos/index
---
2014/10/29 18:01:00 [error] [exception.CDbException] exception 'CDbException' with message 'Relation "pessoa" is not defined in active record class "Requisitos".' in C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php:226
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(296): CActiveFinder->buildJoinTree(Object(CJoinElement), 'pessoa')
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(48): CActiveFinder->buildJoinTree(Object(CJoinElement), Array)
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(909): CActiveFinder->__construct(Object(Requisitos), Array)
#3 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1604): CActiveRecord->getActiveFinder(Array)
#4 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#5 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#6 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#7 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#8 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#9 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(339): CBaseListView->init()
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#12 C:\xampp\htdocs\projman\protected\views\requisitos\admin.php(64): CBaseController->widget('zii.widgets.gri...', Array)
#13 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#17 C:\xampp\htdocs\projman\protected\controllers\RequisitosController.php(144): CController->render('admin', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): RequisitosController->actionAdmin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#22 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#23 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#24 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#25 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#26 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#27 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#28 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/admi...')
#29 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#30 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#31 {main}
REQUEST_URI=/projman/index.php?r=requisitos/admin
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos/index
---
2014/10/29 18:01:01 [error] [exception.CDbException] exception 'CDbException' with message 'Relation "pessoa" is not defined in active record class "Requisitos".' in C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php:226
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(296): CActiveFinder->buildJoinTree(Object(CJoinElement), 'pessoa')
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(48): CActiveFinder->buildJoinTree(Object(CJoinElement), Array)
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(909): CActiveFinder->__construct(Object(Requisitos), Array)
#3 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1604): CActiveRecord->getActiveFinder(Array)
#4 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#5 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#6 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#7 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#8 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#9 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(339): CBaseListView->init()
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#12 C:\xampp\htdocs\projman\protected\views\requisitos\admin.php(64): CBaseController->widget('zii.widgets.gri...', Array)
#13 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#17 C:\xampp\htdocs\projman\protected\controllers\RequisitosController.php(144): CController->render('admin', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): RequisitosController->actionAdmin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#22 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#23 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#24 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#25 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#26 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#27 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#28 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/admi...')
#29 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#30 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#31 {main}
REQUEST_URI=/projman/index.php?r=requisitos/admin
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos/index
---
2014/10/29 18:01:08 [error] [exception.CDbException] exception 'CDbException' with message 'Relation "pessoa" is not defined in active record class "Requisitos".' in C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php:226
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(296): CActiveFinder->buildJoinTree(Object(CJoinElement), 'pessoa')
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(48): CActiveFinder->buildJoinTree(Object(CJoinElement), Array)
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(909): CActiveFinder->__construct(Object(Requisitos), Array)
#3 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1604): CActiveRecord->getActiveFinder(Array)
#4 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#5 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#6 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#7 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#8 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#9 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(339): CBaseListView->init()
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#12 C:\xampp\htdocs\projman\protected\views\requisitos\admin.php(64): CBaseController->widget('zii.widgets.gri...', Array)
#13 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#17 C:\xampp\htdocs\projman\protected\controllers\RequisitosController.php(144): CController->render('admin', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): RequisitosController->actionAdmin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#22 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#23 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#24 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#25 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#26 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#27 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#28 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/admi...')
#29 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#30 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#31 {main}
REQUEST_URI=/projman/index.php?r=requisitos/admin
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos/index
---
2014/10/29 18:01:09 [error] [exception.CDbException] exception 'CDbException' with message 'Relation "pessoa" is not defined in active record class "Requisitos".' in C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php:226
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(296): CActiveFinder->buildJoinTree(Object(CJoinElement), 'pessoa')
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(48): CActiveFinder->buildJoinTree(Object(CJoinElement), Array)
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(909): CActiveFinder->__construct(Object(Requisitos), Array)
#3 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1604): CActiveRecord->getActiveFinder(Array)
#4 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#5 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#6 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#7 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#8 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#9 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(339): CBaseListView->init()
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#12 C:\xampp\htdocs\projman\protected\views\requisitos\admin.php(64): CBaseController->widget('zii.widgets.gri...', Array)
#13 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#17 C:\xampp\htdocs\projman\protected\controllers\RequisitosController.php(144): CController->render('admin', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): RequisitosController->actionAdmin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#22 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#23 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#24 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#25 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#26 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#27 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#28 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/admi...')
#29 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#30 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#31 {main}
REQUEST_URI=/projman/index.php?r=requisitos/admin
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos/index
---
2014/10/29 18:21:07 [error] [exception.CException] exception 'CException' with message 'Property "CDataColumn.label" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(220): CComponent->__set('label', 'Descri\xC3\xA7\xC3\xA3o de ...')
#1 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(388): YiiBase::createComponent(Array, Object(CGridView))
#2 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(359): CGridView->initColumns()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#5 C:\xampp\htdocs\projman\protected\views\requisitos\admin.php(66): CBaseController->widget('zii.widgets.gri...', Array)
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#10 C:\xampp\htdocs\projman\protected\controllers\RequisitosController.php(144): CController->render('admin', Array)
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): RequisitosController->actionAdmin()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#21 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/admi...')
#22 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#23 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/projman/index.php?r=requisitos/admin
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos/index
---
2014/10/29 18:37:32 [error] [exception.CDbException] exception 'CDbException' with message 'The relation "requisitoses0" in active record class "AlteracoesRequisito" is specified with an invalid foreign key "requisitos". There is no such column in the table "alteracoes_requisito".' in C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php:1096
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(1067): CJoinElement->joinOneMany(Object(CJoinElement), Array, Object(CJoinElement), Object(CJoinElement))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(1295): CJoinElement->getJoinCondition()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(790): CJoinQuery->join(Object(CJoinElement))
#3 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(728): CJoinElement->buildQuery(Object(CJoinQuery))
#4 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(151): CJoinElement->count(Object(CDbCriteria))
#5 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1605): CActiveFinder->count(Object(CDbCriteria))
#6 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#7 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#8 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#9 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#10 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#11 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(339): CBaseListView->init()
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#13 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#14 C:\xampp\htdocs\projman\protected\views\alteracoesRequisito\admin.php(78): CBaseController->widget('zii.widgets.gri...', Array)
#15 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#17 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#18 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#19 C:\xampp\htdocs\projman\protected\controllers\AlteracoesRequisitoController.php(109): CController->render('admin', Array)
#20 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): AlteracoesRequisitoController->actionAdmin()
#21 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#22 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#23 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#25 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracoesrequi...')
#26 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#27 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/projman/index.php?r=alteracoesrequisito/admin
HTTP_REFERER=http://localhost/projman/index.php?r=alteracoesrequisito/index
---
2014/10/29 18:38:13 [error] [php] include(Requisito.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:427)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(395): spl_autoload_call()
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(182): model()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(230): CActiveFinder->getModel()
#3 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(296): CActiveFinder->buildJoinTree()
#4 C:\xampp\htdocs\yii\framework\db\ar\CActiveFinder.php(48): CActiveFinder->buildJoinTree()
#5 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(909): CActiveFinder->__construct()
#6 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1604): AlteracoesRequisito->getActiveFinder()
#7 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(228): AlteracoesRequisito->count()
#8 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#9 C:\xampp\htdocs\yii\framework\web\CActiveDataProvider.php(178): CActiveDataProvider->getTotalItemCount()
#10 C:\xampp\htdocs\yii\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#11 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(111): CActiveDataProvider->getData()
#12 C:\xampp\htdocs\yii\framework\zii\widgets\grid\CGridView.php(339): CGridView->init()
#13 C:\xampp\htdocs\yii\framework\web\CBaseController.php(147): CGridView->init()
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): AlteracoesRequisitoController->createWidget()
#15 C:\xampp\htdocs\projman\protected\views\alteracoesRequisito\admin.php(78): AlteracoesRequisitoController->widget()
#16 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#17 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): AlteracoesRequisitoController->renderInternal()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(869): AlteracoesRequisitoController->renderFile()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(782): AlteracoesRequisitoController->renderPartial()
#20 C:\xampp\htdocs\projman\protected\controllers\AlteracoesRequisitoController.php(109): AlteracoesRequisitoController->render()
#21 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): AlteracoesRequisitoController->actionAdmin()
#22 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#23 C:\xampp\htdocs\yii\framework\web\CController.php(286): AlteracoesRequisitoController->runAction()
#24 C:\xampp\htdocs\yii\framework\web\CController.php(265): AlteracoesRequisitoController->runActionWithFilters()
#25 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): AlteracoesRequisitoController->run()
#26 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#27 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#28 C:\xampp\htdocs\projman\index.php(13): CWebApplication->run()
REQUEST_URI=/projman/index.php?r=alteracoesrequisito/admin
in C:\xampp\htdocs\projman\protected\views\alteracoesRequisito\admin.php (78)
in C:\xampp\htdocs\projman\protected\controllers\AlteracoesRequisitoController.php (109)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/02 14:51:57 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'root.tbl_users' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_users`.
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php (27)
in C:\xampp\htdocs\projman\protected\modules\user\models\UserLogin.php (52)
2014/11/02 14:51:57 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(212): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): CActiveRecord->__call('notsafe', Array)
#3 C:\xampp\htdocs\projman\protected\modules\user\components\UserIdentity.php(27): User->notsafe()
#4 C:\xampp\htdocs\projman\protected\modules\user\models\UserLogin.php(52): UserIdentity->authenticate()
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): UserLogin->authenticate('password', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(UserLogin), 'password')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(UserLogin), NULL)
#8 C:\xampp\htdocs\projman\protected\modules\user\controllers\LoginController.php(19): CModel->validate()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/projman/index.php?r=user/login
HTTP_REFERER=http://localhost/projman/index.php?r=user/login
---
2014/11/02 14:53:14 [error] [exception.CException] exception 'CException' with message 'Alias "ext.web-user-behavior.WebUserBehavior" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('ext.web-user-be...', true)
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(329): YiiBase::createComponent('ext.web-user-be...')
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(298): CComponent->attachBehavior(0, 'ext.web-user-be...')
#3 C:\xampp\htdocs\yii\framework\base\CApplicationComponent.php(45): CComponent->attachBehaviors(Array)
#4 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(198): CApplicationComponent->init()
#5 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CWebUser->init()
#6 C:\xampp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('user')
#7 C:\xampp\htdocs\projman\protected\modules\user\controllers\LoginController.php(12): CModule->__get('user')
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=user/login
HTTP_REFERER=http://localhost/projman/index.php?r=user/login
---
2014/11/05 18:39:06 [error] [exception.CException] exception 'CException' with message 'Alias "zii.widgets.CButton" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(146): YiiBase::import('zii.widgets.CBu...', true)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(PessoaController), 'zii.widgets.CBu...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.CBu...', Array)
#3 C:\xampp\htdocs\projman\protected\views\pessoa\admin.php(63): CBaseController->widget('zii.widgets.CBu...', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\projman\protected\controllers\PessoaController.php(144): CController->render('admin', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PessoaController->actionAdmin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('pessoa/admin')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/projman/index.php?r=pessoa/admin
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/index
---
2014/11/05 18:39:44 [error] [exception.CException] exception 'CException' with message 'Property "CJuiButton.icon" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('icon', 'fa fa-print')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(PessoaController), 'zii.widgets.jui...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.jui...', Array)
#3 C:\xampp\htdocs\projman\protected\views\pessoa\admin.php(63): CBaseController->widget('zii.widgets.jui...', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\projman\protected\controllers\PessoaController.php(144): CController->render('admin', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PessoaController->actionAdmin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('pessoa/admin')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/projman/index.php?r=pessoa/admin
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/index
---
2014/11/05 18:39:56 [error] [exception.CException] exception 'CException' with message 'Property "CJuiButton.label" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('label', 'Export')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(PessoaController), 'zii.widgets.jui...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.jui...', Array)
#3 C:\xampp\htdocs\projman\protected\views\pessoa\admin.php(63): CBaseController->widget('zii.widgets.jui...', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\projman\protected\controllers\PessoaController.php(144): CController->render('admin', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PessoaController->actionAdmin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('pessoa/admin')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/projman/index.php?r=pessoa/admin
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/index
---
2014/11/05 18:40:09 [error] [exception.CException] exception 'CException' with message 'Property "CJuiButton.type" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('type', 'primary')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(PessoaController), 'zii.widgets.jui...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.jui...', Array)
#3 C:\xampp\htdocs\projman\protected\views\pessoa\admin.php(63): CBaseController->widget('zii.widgets.jui...', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\projman\protected\controllers\PessoaController.php(144): CController->render('admin', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PessoaController->actionAdmin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('pessoa/admin')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/projman/index.php?r=pessoa/admin
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/index
---
2014/11/05 18:40:20 [error] [exception.CException] exception 'CException' with message 'CJuiButton must specify "model" and "attribute" or "name" property values.' in C:\xampp\htdocs\yii\framework\zii\widgets\jui\CJuiInputWidget.php:59
Stack trace:
#0 C:\xampp\htdocs\yii\framework\zii\widgets\jui\CJuiButton.php(105): CJuiInputWidget->resolveNameID()
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(173): CJuiButton->run()
#2 C:\xampp\htdocs\projman\protected\views\pessoa\admin.php(63): CBaseController->widget('zii.widgets.jui...', Array)
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#7 C:\xampp\htdocs\projman\protected\controllers\PessoaController.php(144): CController->render('admin', Array)
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PessoaController->actionAdmin()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('pessoa/admin')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/projman/index.php?r=pessoa/admin
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/index
---
2014/11/05 18:44:12 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.widgets.TbButton" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\yii\framework\YiiBase.php:347
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(146): YiiBase::import('bootstrap.widge...', true)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(PessoaController), 'bootstrap.widge...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(172): CBaseController->createWidget('bootstrap.widge...', Array)
#3 C:\xampp\htdocs\projman\protected\views\pessoa\admin.php(63): CBaseController->widget('bootstrap.widge...', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\projman\protected\controllers\PessoaController.php(144): CController->render('admin', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PessoaController->actionAdmin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('pessoa/admin')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/projman/index.php?r=pessoa/admin
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/view&id=
---
2014/11/05 18:54:35 [error] [exception.CException] exception 'CException' with message 'Alias "aliased.path.to.booster.directory.and.inside.it.Bootstrap.class" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\yii\framework\YiiBase.php:347
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('aliased.path.to...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(386): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(523): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(163): CModule->preloadComponents()
#4 C:\xampp\htdocs\yii\framework\YiiBase.php(125): CApplication->__construct('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\YiiBase.php(98): YiiBase::createApplication('CWebApplication', 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\projman\index.php(13): YiiBase::createWebApplication('C:\\xampp\\htdocs...')
#7 {main}
REQUEST_URI=/projman/index.php?r=pessoa/admin
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/view&id=
---
2014/11/05 18:58:03 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'. The SQL statement executed was: INSERT INTO `pessoa` (`nome`, `morada`, `tlm`, `data_nascimento`) VALUES (:yp0, :yp1, :yp2, :yp3).
in C:\xampp\htdocs\projman\protected\controllers\PessoaController.php (74)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 18:58:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'. The SQL statement executed was: INSERT INTO `pessoa` (`nome`, `morada`, `tlm`, `data_nascimento`) VALUES (:yp0, :yp1, :yp2, :yp3)' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\projman\protected\controllers\PessoaController.php(74): CActiveRecord->save()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PessoaController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('pessoa/create')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projman/index.php?r=pessoa/create
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/create
---
2014/11/05 19:00:01 [error] [exception.CException] exception 'CException' with message 'PermissaoController cannot find the requested view "index".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#1 C:\xampp\htdocs\projman\protected\controllers\PermissaoController.php(129): CController->render('index', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PermissaoController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('permissao/index')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/projman/index.php?r=permissao/index
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/index
---
2014/11/05 19:00:42 [error] [exception.CException] exception 'CException' with message 'PermissaoController cannot find the requested view "index".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#1 C:\xampp\htdocs\projman\protected\controllers\PermissaoController.php(129): CController->render('index', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PermissaoController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('permissao/index')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/projman/index.php?r=permissao/index
HTTP_REFERER=http://localhost/projman/index.php?r=pessoa/index
---
2014/11/05 19:24:20 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 19:24:49 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 19:25:02 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 19:26:05 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "RequisitosController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(69): CInlineFilter::create(Object(RequisitosController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(RequisitosController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/crea...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=requisitos/create
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos
---
2014/11/05 19:28:25 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "RequisitosController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(69): CInlineFilter::create(Object(RequisitosController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(RequisitosController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/crea...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=requisitos/create
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos
---
2014/11/05 19:28:29 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "RequisitosController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(69): CInlineFilter::create(Object(RequisitosController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(RequisitosController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/admi...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=requisitos/admin
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos
---
2014/11/05 19:58:05 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 19:58:45 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "RequisitosController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(69): CInlineFilter::create(Object(RequisitosController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(RequisitosController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/crea...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=requisitos/create
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos
---
2014/11/05 20:27:58 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "RequisitosController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(72): CInlineFilter::create(Object(RequisitosController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(RequisitosController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/crea...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=requisitos/create
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos
---
2014/11/05 20:28:58 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "RequisitosController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(69): CInlineFilter::create(Object(RequisitosController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(RequisitosController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('requisitos/crea...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=requisitos/create
HTTP_REFERER=http://localhost/projman/index.php?r=requisitos
---
2014/11/05 20:32:17 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:32:21 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:32:26 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "StakeholderController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(69): CInlineFilter::create(Object(StakeholderController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(StakeholderController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('stakeholder/adm...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=stakeholder/admin
HTTP_REFERER=http://localhost/projman/index.php?r=stakeholder
---
2014/11/05 20:33:14 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:33:18 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:35:21 [warning] [application] Failed to set unsafe attribute "cod_projecto" of "Projecto".
in C:\xampp\htdocs\projman\protected\controllers\ProjectoController.php (263)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:36:50 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:36:56 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:40:16 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:40:19 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:41:15 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:41:22 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:41:26 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:43:33 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:43:37 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:44:19 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:44:22 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:44:51 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:44:54 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:45:31 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:45:34 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:47:12 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:47:15 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:47:39 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:47:42 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/05 20:51:09 [error] [exception.CException] exception 'CException' with message 'Filter "rights" is invalid. Controller "AlteracaoController" does not have the filter method "filterrights".' in C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php:46
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(69): CInlineFilter::create(Object(AlteracaoController), 'rights')
#1 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain::create(Object(AlteracaoController), Object(CInlineAction), Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracao/creat...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/projman/index.php?r=alteracao/create
HTTP_REFERER=http://localhost/projman/index.php?r=alteracao/calendar
---
2014/11/05 21:02:50 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php:62
Stack trace:
#0 C:\xampp\htdocs\projman\protected\modules\rights\components\RightsFilter.php(57): RController->accessDenied()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php(36): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracao/creat...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=alteracao/create
HTTP_REFERER=http://localhost/projman/index.php?r=alteracao/index
---
2014/11/07 20:35:18 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php:62
Stack trace:
#0 C:\xampp\htdocs\projman\protected\modules\rights\components\RightsFilter.php(57): RController->accessDenied()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php(36): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracao/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=alteracao/admin
HTTP_REFERER=http://localhost/projman/index.php?r=user/login
---
2014/11/14 01:38:20 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php:62
Stack trace:
#0 C:\xampp\htdocs\projman\protected\modules\rights\components\RightsFilter.php(57): RController->accessDenied()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php(36): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('casouso/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=casouso/admin
HTTP_REFERER=http://localhost/projman/index.php?r=casouso/index
---
2014/11/16 14:08:32 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php:62
Stack trace:
#0 C:\xampp\htdocs\projman\protected\modules\rights\components\RightsFilter.php(57): RController->accessDenied()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): RightsFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\projman\protected\modules\rights\components\RController.php(36): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): RController->filterRights(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('alteracoesrequi...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/projman/index.php?r=alteracoesrequisito/admin
HTTP_REFERER=http://localhost/projman/index.php?r=alteracoesrequisito/index
---
2014/11/16 14:32:09 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/16 14:32:49 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/16 14:34:15 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/16 14:34:39 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/16 14:36:22 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/16 14:43:18 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/16 14:43:32 [warning] [application] Failed to set unsafe attribute "file" of "CrudCode".
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:49:46 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it.
' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\web\auth\CDbAuthManager.php(589): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\web\auth\CDbAuthManager.php(59): CDbAuthManager->getDbConnection()
#5 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbAuthManager->init()
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(185): CModule->getComponent('authManager')
#7 C:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(815): CWebApplication->getAuthManager()
#8 C:\xampp\htdocs\projman\protected\modules\rights\components\RWebUser.php(43): CWebUser->checkAccess('Admin', Array, true)
#9 C:\xampp\htdocs\projman\protected\views\layouts\main.php(39): RWebUser->checkAccess('Admin')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#13 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#14 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#15 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#16 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#17 C:\xampp\htdocs\projman\protected\views\layouts\column1.php(6): CBaseController->endContent()
#18 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#19 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#20 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#21 C:\xampp\htdocs\projman\protected\controllers\SiteController.php(32): CController->render('index')
#22 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#23 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#24 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#25 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#26 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#27 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#28 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#29 C:\xampp\htdocs\projman\index.php(13): CApplication->run()
#30 {main}
REQUEST_URI=/projman/
---
2014/11/21 10:51:32 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:51:38 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:51:43 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:51:53 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:51:57 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:52:01 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:52:10 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:53:36 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:53:43 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:53:47 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:53:58 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:54:02 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:54:04 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
2014/11/21 10:54:11 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\projman\protected\modules\user\controllers\RegistrationController.php (30)
in C:\xampp\htdocs\projman\index.php (13)
